src/main/java/com/frcteam3636/frc2024/subsystems/shooter/Shooter.kt

```diff
@@ -9,9 +9,17 @@ import edu.wpi.first.wpilibj2.command.Command
 import edu.wpi.first.wpilibj2.command.PIDCommand
 import edu.wpi.first.wpilibj2.command.Subsystem
 import org.littletonrobotics.junction.Logger
+import com.ctre.phoenix6.controls.DynamicMotionMagicTorqueCurrentFOC
+import edu.wpi.first.wpilibj2.command.InstantCommand
+import edu.wpi.first.math.geometry.Rotation2d
 
 object Shooter : Subsystem {
-    private val io: ShooterIO =
+
+    const val ACCELERATION_PROFILE = 0.0
```

Comment:
These should probably be in a `Constants` companion object like we do elsewhere.

Reply:
i can't put it in a companion object cause shooter is an object not a class